src/main/java/com/ibm/cloud/cloudant/kafka/tasks/SourceChangesTask.java

```diff
@@ -50,6 +48,11 @@ public class SourceChangesTask extends org.apache.kafka.connect.source.SourceTas
 
     private List<String> topics = null;
 
+    // Note this is a unique identifier for the Cloudant source.
+    // At present we consider a DB the source partition and
+    // uniquely identify it by URL and name.
+    private Map<String, String> sourcePartition = new HashMap<>(2);
```

Comment:
This field could be turned into a local variable since it is only used in `start`

Reply:
good spot, done in f182320